src/swati/Ass23FindCase.java

```diff
@@ -0,0 +1,35 @@
+/*Assignment - 23 : 14th Sep'2023 Write a program to find count of Uppercase, lowercase, digits and special characters. String str = "TecHNo_cR6ediTs"; output : 
+Uppercase -> 5  
+Lowercase -> 8 
+Digit -> 1 Special
+char count -> 1*/
+
+package swati;
+
+public class Ass23FindCase {
+	int uppercasecount = 0;
+	int lowercasecount = 0;
+	int digitcount = 0;
+	int specialcount = 0;
+
+	void findCharCase(String input) {
+		for (int index = 0; index < input.length(); index++) {
+			if (Character.isUpperCase(input.charAt(index))) {
```

Comment:
It would be better if you create char ch = str.charAt(index) outside of condition, so you no need to write it again and again, you have to just write ch